src/containers/Alerts/index.js

```diff
@@ -41,28 +43,32 @@ function AlertsComponent(props) {
 
 	return (
 		<div className="alerts" >
-			{props.alerts.map((alert) => {
+			{alerts.map((alert) => {
 				return (
 					<Alert
 						key={alert.key}
 						color={alert.level}
 						className="shadow alerts-style"
 						fade={true}
 						isOpen={!alert.acked}
-						toggle={() => store.dispatch({ type: ACK_ALERT, key: alert.key })}
+						toggle={!sessionExpired ? () => store.dispatch({ type: ACK_ALERT, key: alert.key }) : null} // remove the close button for alert with expiration
 					>
 						{alert.shouldBeTranslated ? t(alert.message) : alert.message}
+						{" "}
+						{sessionExpired &&
+							<a
+								className="alert-link"
+								onClick={() => window.location.reload(false)} // refreshes the page
+								href="#"
+								rel="noreferrer"
+								target="_blank"
+							>
+								{t("Alerts|here")}
+							</a>
+						}
```

Comment:
@Pe5h4 I'm not sure about these changes. Do you think it's right or not?

https://user-images.githubusercontent.com/65866093/229820535-fec0b2d9-5c75-49b2-8b20-e3af67a002a3.mp4

Reply:
@Aringoaway You know what? You dont need to use `a` tag at all. Why cant you use just
```
{sessionExpired &&
	<span onClick={window.location.reload} className="alert-span">
		{` ${t("Alerts|here")}`}
	</span>
}
```
and style the span with cursor and maybe some other styles?

Your approach seems to me bit like a overkill :))